dashboard/public/pixelation/pixelation.html

```diff
@@ -36,7 +36,7 @@
       </td><td>
         <input type="number" min="1" max="255" value="10" id="width" oninput="setSliders();">
       </td><td class="hide_on_v2">
-        <input type="range" min="1" max="255" value="10" id="widthRange" oninput="changeVal('width');" class="rangeInput">
+        <input type="range" min="1" max="255" value="10" id="widthRange" oninput="changeVal('width');" onchange="changeVal('width');" class="rangeInput">
```

Comment:
Does `changeVal` fire twice now on newer browsers?  This might be a good candidate for a `debounce` or `throttle`, not just to avoid calling it twice here but for performance purposes in general.